homeassistant/components/isy994/light.py

```diff
@@ -76,14 +80,21 @@ def turn_off(self, **kwargs) -> None:
     def on_update(self, event: object) -> None:
         """Save brightness in the update event from the ISY994 Node."""
         if self._node.status not in (0, ISY_VALUE_UNKNOWN):
-            self._last_brightness = self._node.status
+            self._last_brightness = (
+                self._node.status
+                if self._node.uom != UOM_PERCENTAGE
```

Comment:
clarify: If the ternary spans multiple lines, the Home Assistant standard is to unpack it into an `if`

Reply:
Updated.

Reply:
Thanks